### PR TITLE
Added dates for all releases to changelog

### PR DESCRIPTION
**Pull request by cmelchior**
Changelog has been cleaned up, so now all releases contain the release data instead of just the first 4.

@realm/java

### CODE REVIEW

**Comment by emanuelez** [comment]
:-1:  we lose the ability to automate releases. :(

**Comment by cmelchior** [comment]
Seems relatively easy to create a search/replace regexp, especially from Groovy?

File: changelog.txt [hunk 1]
```diff
@@ -89,14 +89,14 @@
  * Solved ConcurrentModificationException thrown when addChangeListener/removeChangeListener got called in the onChange. (thanks @beeender)
  * Fixed duplicated listeners in the same realm instance. Trying to add duplicated listeners is ignored now. (thanks @beeender)
 
-0.80.2
+0.80.2 (4 Maj 2015)
```
Comment (kneth) [inline changelog.txt]: May

**Comment by kneth** [comment]
I do like the idea so you get :+1: from me.

**Comment by dalinaum** [comment]
:+1:

**Comment by emanuelez** [comment]
Not with the release process of the new build system, unless we decide to fork the plugin we are using.

**Comment by cmelchior** [comment]
I'm fine with closing this. I think it is better to have the dates there, but it is a minor itch. I'll find some other way to scratch it :)

**Comment by bmunkholm** [comment]
I've been missing the dates as well. I think we should include them. It's totally fine to not have the date there automatically, we can add that manually even after the release (just like it's been done now).

**Pull request closed without merging**